Fixes #7615 - filter ignore current context

ares opened the pull request.

ares committed:
Fixes #7615 - filter ignore current context

dLobatog commented on test/unit/filter_test.rb:
```diff
@@ -90,6 +90,14 @@ class FilterTest < ActiveSupport::TestCase
       end
     end
 
+    test 'filter is not automatically scoped to to taxonomies' do
```
Typo in the test description, `scoped to any` taxonomies.

In case you want to save the variable assignment (original_org) in the test:

```
@organization, Organization.current = Organization.current, @organization
assert_empty Filter.new.organizations
Organization.current, @organization = @organization, Organization.current
```

dLobatog commented:
@ares :+1: by the way, thanks. I'll merge after fixing the typo at least.

ares commented:
good catch, fixed the typo and changed the assignment a bit, I don't want to touch the instance variable that is shared for other tests but the multiple assignment on first line is good trick to suggest that we're saving the original value because of Organization.current reassignment

dLobatog commented:
Jenkins said: "Could not find gem 'rails (= 3.2.21) ruby' in the gems available on this", weird. [test]

domcleal commented:
> Jenkins said: "Could not find gem 'rails (= 3.2.21) ruby' in the gems available on this", weird. [test]

The actual error is earlier in the log, that is just from the cleanup routine which fails as `bundle install` had failed (due to execjs, now fixed).

dLobatog commented:
Merged as 76fca93b0093e9b4174d1a888ffca9f3fcb886c9, thanks @ares
